sklearn/svm/_base.py

```diff
@@ -677,14 +680,18 @@ def n_support_(self):
             check_is_fitted(self)
         except NotFittedError:
             raise AttributeError
+        return self._n_support
```

Comment:
I think it's better to move this into `BaseSVC`. This way when `n_support_` is removed, then we only need to remove the `@property` from the regressors.